packages/sitecore-jss/src/layout/utils.test.ts

```diff
@@ -181,6 +181,22 @@ describe('sitecore-jss layout utils', () => {
         const result = isFieldValueEmpty(field);
         expect(result).to.be.false;
       });
+
+      it('should return true if Date field is empty for Date type', () => {
+        expect(
+          isFieldValueEmpty({
+            value: undefined,
+          })
+        ).to.be.true;
+      });
+
+      it('should return false if Date field is not empty for Date type', () => {
```

Comment:
Should we add a test case for invalid date object?

Reply:
makes sense, added